GHzDACs/ghz_fpga_server.py

```diff
@@ -850,8 +850,8 @@ def recoverFromTimeout(self, runners, results):
                     )
                 runner.executionCount = count
             except Exception as e:
+                print("Attempt to read execution counts failed: ")
```

Comment:
We should be consistent in using or not using parens with print. I like doing `from __future__ import print_function` and using parens everywhere, but we should probably make that change in one go for the whole file. Until then, omit.